docs/UserGuide.md

```diff
@@ -48,12 +48,13 @@ Format: `add n/[item_name] upc/[UPC] qty/[quantity] p/[price] c/[category]`
 * The `p/` parameter for `[price]` must be a **non-negative numerical** value _(decimals accepted)_.
 
 **OPTIONAL** parameters:
-* The `c/` parameter for `[category]` must be alphanumeric. (Defaults to: `Uncategorized` if not specified.)
```

Comment:
ps i added this "if not specified" because one of the issue closed previously mentioned that user may not be aware of what will happen if c/ is not provided. Hence, i added this for clarity just curious why it is removed

Reply:
Sorry merge conflicts issue again